db/migrate/20150625153556_add_dept_and_status_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddDeptAndStatusToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :ucstatus, :string
+    add_column :users, :ucdepartment, :string
```

Comment:
It was a part of the migration that I copied from Scholar. It's not used anywhere, if you think it should be removed I can do that.